csvcubed/tests/unit/cube/qb/test_cubeqb_errorvalidation.py

```diff
@@ -123,6 +123,9 @@ def test_existing_dimension_csv_column_uri_template():
     assert isinstance(validation_error, CsvColumnUriTemplateMissingError)
     assert validation_error.csv_column_name == "Existing Dimension"
 
+    assert isinstance(CsvColumnUriTemplateMissingError.get_error_url(), str)
+    assert CsvColumnUriTemplateMissingError.get_error_url() == 'http://purl.org/csv-cubed/err/csv-col-uri-temp-mis'
```

Comment:
It'd make me feel more comfortable to use `validation_error.get_error_url()` here since we need the error URL to be linked to the particular error instance.

Reply:
Same in all the other locations.